text/0038-output-types-for-tokenization-and-sc/0038-output-types-for-tokenization-and-sc.md

```diff
@@ -0,0 +1,3416 @@
+                            </tr>
+                        </table>
+                    </details>
+                    <details>
+                        <summary>NFT Address</summary>
+                        <table>
+                            <tr>
+                                <td><b>Name</b></td>
+                                <td><b>Type</b></td>
+                                <td><b>Description</b></td>
+                            </tr>
+                            <tr>
+                                <td>Address Type</td>
+                                <td>uint8</td>
+                                <td>
+                                    Set to <strong>value 16</strong> to denote an <i>NFT Address</i>.
+                                </td>
+                            </tr>
+                            <tr>
+                                <td>Address</td>
+                                <td>ByteArray[20]</td>
+                                <td>The raw bytes of the nft address which is a BLAKE2b-160 hash of the outputID that created it.</td>
+                            </tr>
+                        </table>
+                    </details>
+                </td>
+            </tr>
+            <tr>
+                <td>Amount</td>
+                <td>uint64</td>
+                <td>The amount of IOTA coins to held by the output.</td>
+            </tr>
+            <tr>
+                <td>Native Token Count</td>
+                <td>uint16</td>
+                <td>The number of native tokens held by the output.</td>
+            </tr>
+            <tr>
+                <td valign="top">Native Tokens <code>optAnyOf</code></td>
+                <td colspan="2">
+                    <details>
+                        <summary>Native Token</summary>
+                        <table>
+                            <tr>
+                                <td><b>Name</b></td>
+                                <td><b>Type</b></td>
+                                <td><b>Description</b></td>
+                            </tr>
+                            <tr>
+                                <td>Token ID</td>
+                                <td>ByteArray[38]</td>
+                                <td>
+                                    Identifier of the native token.
+                                </td>
+                            </tr>
+                            <tr>
+                                <td>Amount</td>
+                                <td>uint256</td>
+                                <td>
+                                    Amount of native tokens of the given <i>Token ID</i>.
+                                </td>
+                            </tr>
+                        </table>
+                    </details>
+                </td>
+            </tr>
+            <tr>
+                <td>Blocks Count</td>
+                <td>uint16</td>
```

Comment:
Wouldn't uint8 be enough?

Reply:
Updated `Blocks Count` to uint8, as with the `atMostOneOfEach` rule and `Block Type` being uint8, we can't really have more than 255 feature blocks.